inductiva/tasks/task.py

```diff
@@ -152,25 +157,91 @@ def get_output(self, output_dir=None):
         Returns:
             The output of the task.
         """
-
-        # Blocking call for the task to terminate
         self.wait()
-        _, output_dir = api.download_output(self._api, self.id, output_dir)
-        output_dir = files.resolve_path(output_dir)
+        output_dir = self.download_outputs(output_dir)
 
         if self._output_class:
             return self._output_class(output_dir)
 
         return output_dir
 
-    def download_output(self, output_dir=None) -> pathlib.Path:
-        """Download the output of the task.
+    def get_outputs_info(self):
+        """Get information of the output files of the task.
```

Comment:
I think we need to think about this name `get_outputs_info` since with this change the `get_output` may just fetch some files of the output for each scenario.

Reply:
`get_output_files_info` ?